app/src/androidTest/java/com/sdp/movemeet/viewTest/navigationTest/ChatActivityNavigationTest.java

```diff
@@ -51,7 +51,7 @@ public void signIn(){
         // Open Drawer to click on navigation.
         onView(withId(R.id.drawer_layout)).check(matches(isClosed(Gravity.LEFT))).perform(DrawerActions.open());
         try{
-            Thread.sleep(500);
+            Thread.sleep(2500);
```

Comment:
Yes, that's strange because I didn't have this kind of problem before but this time, I seems that that drawer didn't entirely load in 500ms. I admit that 2,5s is maybe a bit too much xD

Reply:
Yes probably... Our Thursday evening merge sessions will end even later then! But no problem if this allows the tests to pass! 😉